src/Event/ReplayHeadersEvent.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+
+/**
+ * Class ReplayHeadersEvent
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+class ReplayHeadersEvent extends Event
+{
+    const EVENT_NAME = 'fos.replay_headers';
```

Comment:
the event name should start with ``fos_http_cache`` instead. We have more bundles in FOS than this one. The bundle config prefix is not just ``fos``